packages/ui5-info/jest.config.js

```diff
@@ -0,0 +1,7 @@
+const config = require('../../jest.base');
+config.snapshotFormat = {
+    escapeString: false,
+    printBasicPrototype: false
+};
```

Comment:
```suggestion
```
I think false is the default so this can be removed.

Reply:
The defaults are true in both cases : https://github.com/SAP/open-ux-tools/blob/main/jest.base.js#L25 so it required to change the behaviour. IMO the defaults are leading to hard to read and bloated snapshots.